src/client/pages/OfferNew/Introduction/HeroOfferDetails.tsx

```diff
@@ -1,75 +1,84 @@
 import React, { useState, useEffect } from 'react'
 import styled from '@emotion/styled'
 import { colorsV3 } from '@hedviginsurance/brand'
-import { Address } from 'data/graphql'
 import { OfferData } from 'pages/OfferNew/types'
 import { useTextKeys } from 'utils/textKeys'
-import { LARGE_SCREEN_MEDIA_QUERY } from 'utils/mediaQueries'
-import { getHouseholdSize } from '../utils'
+import {
+  LARGE_SCREEN_MEDIA_QUERY,
+  MEDIUM_SCREEN_MEDIA_QUERY,
+} from 'utils/mediaQueries'
+import { getHouseholdSize, quoteDetailsHasAddress } from '../utils'
+import { getAddress } from '../Checkout/InsuranceSummaryDetails'
```

Comment:
Should this function be moved to the more central utils module? Just to make it clear that it is not just used by `InsuranceSummaryDetails` anymore

Reply:
Probably, I'm not sure what approach we should have though. I'm really not a fan of the enormous `utils` files we have in a bunch of places in this repo. Would probably rather have a `/utils` folder or something, with more files with less functions in them. Would make it easier to navigate I think. And also easier to move things around maybe? What do you think?

Reply:
Sounds good to me to split up large utils files to more specific files in a folder. 🙌

Reply:
Yes totally agree with the folder + multiple files approach!

Makes refactoring/deleting things easier.

I just read this article that suggested the same thing:https://programmingisterrible.com/post/139222674273/how-to-write-disposable-code-in-large-systems

I was thinking it would be a good discussion point for a tech weekly or similar actually!